tests/universal/keyexpr.cxx

```diff
@@ -31,21 +29,22 @@ void key_expr_view() {
     assert(foo.as_bytes() == "FOO");
     assert(foo.as_string_view() == "FOO");
 
+#ifdef ZENOHCXX_ZENOHC
```

Comment:
zenohpico is unable to accept string_view as parameter of z_keyexpr because it doesn't have function `zc_keyexpr_from_slice` which don't need trailing zero